src/sirinternal.c

```diff
@@ -55,10 +55,17 @@ static sirconfig _sir_cfg      = {0};
 static sirfcache _sir_fc       = {0};
 static sir_plugincache _sir_pc = {0};
 
+#if !defined(__IMPORTC__)
 static sir_mutex cfg_mutex  = SIR_MUTEX_INIT;
 static sir_mutex fc_mutex   = SIR_MUTEX_INIT;
 static sir_mutex pc_mutex   = SIR_MUTEX_INIT;
 static sir_mutex ts_mutex   = SIR_MUTEX_INIT;
+#else
```

Comment:
just do this where SIR_MUTEX_INIT is defined, silly.

Reply:
That's the bug tho, as discussed.